go/src/koding/kites/kloud/stack/provider/describe.go

```diff
@@ -113,3 +122,30 @@ func mustDescribe(v interface{}) []stack.Value {
 	}
 	return desc
 }
+
+var titler = strings.NewReplacer("_", " ")
+
+func title(s string) string {
```

Comment:
Ping. Merging for now.

I find it harder to reason as in your version there's more "magic" behind regexp functors.

However if you find a use-case that existing impl is not enough, feel encouraged to send PR with your version.